lightwood/api/json_ai.py

```diff
@@ -1180,6 +1180,15 @@ def code_from_json_ai(json_ai: JsonAI) -> str:
 
     predict_body = align(predict_body, 2)
 
+    test_body = """
+predictions = self.predict(data)
+values = [metric(data[self.target], predictions) for metric in metrics]
```

Comment:
`predictions` is a DataFrame that can potentially have many columns. You should pass only the actual predicted values that are contained in the `prediction` column.

Reply:
```suggestion
values = [metric(data[self.target], predictions[self.target]) for metric in metrics]
```